packages/react-components/src/menu/src/MenuItem.tsx

```diff
@@ -74,7 +87,7 @@ export function InnerMenuItem({
         _: {
             defaultWrapper: Text
         },
-        icon: (element, allElements) => {
+        icon: (_element: ReactElement, allElements: Record<string, any>) => {
```

Comment:
Why the "_" ?

Could use the same name suggestion as I suggested in Listbox.

Reply:
In our TSConfig, we added the noUnusedParameter: true.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals

Parameters declaration with names starting with _ are exempt from the unused parameter checking. e.g.:

This rules helps maintain a cleaner codebase, notifying you when you can remove a parameter from a function. In this case, we have to keep it since we need it to match a onChange event handler. but on custom function, it's easier to notify and remove them

Reply:
done for the variable rename